addons/account/models/account_tax_carryover_line.py

```diff
@@ -8,5 +9,21 @@ class AccountTaxCarryoverLine(models.Model):
     name = fields.Char(required=True)
     amount = fields.Float(required=True, default=0.0)
     date = fields.Date(required=True, default=fields.Date.context_today, readonly=True)
-    tax_report_line_id = fields.Many2one(string="Tax report line", comodel_name='account.tax.report.line')
-    company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True, default=lambda self: self.env.company)
+    tax_report_line_id = fields.Many2one(comodel_name='account.tax.report.line', string="Tax report line")
+    tax_report_country_id = fields.Many2one(related='tax_report_line_id.report_id.country_id')
+    company_id = fields.Many2one(comodel_name='res.company', string='Company',
+                                 required=True, readonly=True,
```

Comment:
I wouldn't put the readonly here. People might want to display it for debugging. It's hidden by default anyway.

Reply:
Display has nothing to do with readonly